examples/crd_one_var.R

```diff
@@ -0,0 +1,97 @@
+
+sep <- function(n){
+  # This function simply prints a line of "=" to the screen as a separator and
+  # will not show up in the app. It is only here for nice printing.
+  line <- paste0(paste(replicate(n, "="), collapse = ""), '\n')
+  cat(line)
+}
+
+# The sweetpotato data set is a completely random experiment design with plots
+# of 79 sweet potatoes where there are three viruses and a control applied in 3
+# reps, i.e. 3 plots for each of four virus levels.
+#-----------------------------------------------------------------------------#
+data(sweetpotato)  # load data from agricolae
+my.data <- sweetpotato  # rename the data frame
+#-----------------------------------------------------------------------------#
+
+# Construct the model.
+#-----------------------------------------------------------------------------#
+model <- aov(yield ~ virus, data = my.data)
+#-----------------------------------------------------------------------------#
+
+# Box plot of the effect on virus on yield.
+dev.new()
+#-----------------------------------------------------------------------------#
+boxplot(yield ~ virus, data = my.data, main = "Effect of virus on yield",
+        xlab = "virus", ylab = "yield (?)")
+#-----------------------------------------------------------------------------#
+
+# Plot two standard fit plots: residuals vs predicted, Normal Q-Q plot of the
+# residuals.
+dev.new()
+#-----------------------------------------------------------------------------#
+par(mfrow = c(2, 1), oma = c(0, 0, 2, 0))  # plots as subplots of single graph
+plot(model, which = c(1, 2))
+#-----------------------------------------------------------------------------#
+
+# Make sure the residuals are normal (this can also be seen in the Q-Q plot).
+cat('Shapiro-Wilk Normality Test\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+shapiro.test(residuals(model))
+#-----------------------------------------------------------------------------#
+sep(79)
+
+# Run Levene's Test.
+cat("Levene's Test\n")
+sep(79)
+#-----------------------------------------------------------------------------#
+leveneTest(yield ~ virus, data = my.data)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+#------------------------------------------------------------------------------#
+sep(79)
+
+# Print the ANOVA table of the fit. The user should will have to note the
+# significant factors. In this case the single virus factor is significant.
+cat('ANOVA Table\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+anova(model)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+# Show the confidence intervals for the intercept and each virus level.
+cat('Confidence Intervals\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+confint(model)
```

Comment:
The LSD.test is giving 95% confidence intervals for the least square means of the 9 clone-nitrogen treatment combinations, which is what we want. The other confint() is not. 

For more background, a discussion that's related to this can be found here: http://stats.stackexchange.com/questions/29479/difference-between-ci-of-anova-level-coefficients-vs-t-test-ci-which-one-is-c